FirebaseInstallations/Source/Library/FIRInstallationsItem.h

```diff
@@ -47,6 +48,9 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property(nonatomic, nullable) FIRInstallationsStoredRegistrationError *registrationError;
 
+/// Instance ID checkin data imported from IID checkin store as a part of IID migration.
+@property(nonatomic, nullable) FIRInstallationsStoredIIDCheckin *IIDCheckin;
```

Comment:
My impression is that FIS should not be aware of any checkin data. Especially we are on the plan of removing checkin.

Reply:
talked offline and looks like this is just to read from storage for authentication.